source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md

```diff
@@ -0,0 +1,67 @@
+---
+title: vagrant-service-manager 1.3.0 Released - Focus on Kubernetes Support
+author: budhram
+date: 2016-08-16 12:54:00 UTC
+tags: vagrant, devtools, releases, kubernetes
+published: true
+comments: true
+---
+
+This release of [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager) introduces support for displaying Kubernetes configuration information. This enable users to access the Kubernetes server running inside ADB virtual machine from their host.
+
+Client binary installation support for 'Kubernetes' is included in this release. Support for client binary installation has also been extended to users of the [Red Hat Container Development Kit](http://developers.redhat.com/products/cdk/overview). Information about client binary installation can be found in the previous release announcement ["Client Binary Installation Now Included in the ADB"](../../../../blog/2016/07/vagrant-service-manager-install-cli).
+
+The main new features of this release are:
```

Comment:
If we already list 2 release highlights above, we can change this sentence to say "full list of features". Also, if we're enhancing existing features and not just adding new features, then the word "new" is not necessary.